hwilib/commands.py

```diff
@@ -30,8 +30,7 @@ def get_client(device_type, device_path, password=None):
         device_path = bytes(device_path.encode())
         device.open_path(device_path)
     except Exception as e:
-        print(e)
-        return {'error':'Unable to connect to specified device','code':DEVICE_CONN_ERROR}
+        return {'error':'Unable to connect to specified device:' + str(e),'code':DEVICE_CONN_ERROR}
```

Comment:
not a Pythonese expert, but is there a reason why we can't just throw an error here instead of returning a dict, and handling it as appropriate by the caller?

Reply:
Because that would be the logical thing to do...


Done.